scripts/e2e-custom.sh

```diff
@@ -257,6 +288,31 @@ test_taxonomy() {
     ilab taxonomy diff "${DIFF_ARGS[@]}"
 }
 
+test_convert_taxonomy_for_rag() {
+    step Convert user taxonomy for ingestion into vector store
+    ilab rag convert
```

Comment:
This should be configured to locate the path where `test_taxonomy` puts the taxonomy repo, e.g. `./taxonomy`:
```bash
    ilab rag convert --taxonomy-path taxonomy
```

Reply:
Isn't there a default set in the config? https://github.com/instructlab/instructlab/blob/main/src/instructlab/configuration.py#L301

Reply:
there is, but the script clones taxonomy repo under the current folder, so we need to override it (of course we could update the script to use the default but it looks beyond the scope)